frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx

```diff
@@ -180,19 +145,61 @@ export const InstallPlanTableRow: RowFunction<K8sResourceKind> = ({ obj, index,
   );
 };
 
-export const InstallPlansList = requireOperatorGroup((props: InstallPlansListProps) => {
-  const EmptyMsg = () => (
+const EmptyMsg: React.FC = () => {
+  const { t } = useTranslation();
+  return (
     <MsgBox
-      title="No Install Plans Found"
-      detail="Install Plans are created automatically by subscriptions or manually using the CLI."
+      title={t('olm~No InstallPlans found')}
+      detail={t(
+        'olm~InstallPlans are created automatically by subscriptions or manually using the CLI.',
+      )}
     />
   );
+};
+
+export const InstallPlansList = requireOperatorGroup((props: InstallPlansListProps) => {
+  const { t } = useTranslation();
+  const InstallPlanTableHeader = () => {
+    return [
+      {
+        title: t('olm~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[0] },
+      },
+      {
+        title: t('olm~Namespace'),
+        sortField: 'metadata.namespace',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+      },
+      {
+        title: t('olm~Status'),
+        sortField: 'status.phase',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[2] },
+      },
+      {
+        title: t('olm~Components'),
+        props: { className: tableColumnClasses[3] },
+      },
+      {
+        title: t('olm~Subscriptions'),
+        props: { className: tableColumnClasses[4] },
+      },
+      {
+        title: '',
+        props: { className: tableColumnClasses[5] },
+      },
+    ];
+  };
+
   return (
     <Table
       {...props}
-      aria-label="Install Plans"
+      aria-label={InstallPlanModel.plural}
```

Comment:
Probably we should externalize this

```suggestion
      aria-label={t('olm~InstallPlans')}
```